utils/linkage.py

```diff
@@ -47,3 +48,54 @@ def get_street_from_address_line_1(address_line_1: str) -> str:
             string.append(key)
 
     return " ".join(string)
+
+
+"""
+Module for standardizing the 'company' columnn of the state campaign finance dataset
+"""
+
+
+def cleaning_company_column(company: str) -> str:
+    """
+    Given a string, check if it contains a variation of self employed, unemployed,
+    or retired and return the standardized version.
+
+    Args:
+        company: string of inputted company names
+    Returns:
+        standardized for retired, self employed, and unemployed,
+        or original string if no match or empty string
+
+    >>> cleaning_company_column("Retireed")
+    'Retired'
+    >>> cleaning_company_column("self")
+    'Self Employed'
+    >>> cleaning_company_column("None")
+    'Unemployed'
+    """
+    if pd.isnull(company):
+        return company
+
+    company_edited = company.lower()
+    company_edited = company_edited.strip()
+    company_edited = company_edited.replace(".", " ")
+    company_edited = company_edited.replace(",", " ")
+    company_edited = company_edited.replace("-", " ")
+
+    if "retire" in company_edited:
```

Comment:
let's think about how we can handle people with "retire" in part of their company name